articles/governance/policy/overview.md

```diff
@@ -167,9 +167,9 @@ settings and objects. To find out more, see
 
 Here are a few pointers and tips to keep in mind:
 
-- Start with an audit effect instead of a deny effect to track impact of your policy definition on
+- Start with an audit or auditIfNotExist effect instead of a enforcement (deny, modify, deployIfNotexist) effect to track impact of your policy definition on
```

Comment:
Please surround references to Azure Policy effects with backticks to denote inline code.